app/forms/idv/document_capture_form.rb

```diff
@@ -2,12 +2,17 @@ module Idv
   class DocumentCaptureForm
     include ActiveModel::Model
 
-    ATTRIBUTES = %i[front_image front_image_data_url back_image back_image_data_url].freeze
+    ATTRIBUTES = %i[front_image front_image_data_url
+                    back_image back_image_data_url
+                    selfie_image selfie_image_data_url].freeze
 
-    attr_accessor :front_image, :front_image_data_url, :back_image, :back_image_data_url
+    attr_accessor :front_image, :front_image_data_url,
+                  :back_image, :back_image_data_url,
+                  :selfie_image, :selfie_image_data_url
 
     validate :front_image_or_image_data_url_presence
     validate :back_image_or_image_data_url_presence
+    validate :selfie_image_or_image_data_url_presence
```

Comment:
We can come back around to this in another story if we want, but we are going to want to want a way to disable this is the SP does not require a selfie to proof.